### ISSUE TITLE
[Bug]: Not avalaible for download (v3.2.4)

### ISSUE THREAD

**Issue by tazzkiller** [BlafKing/sd-civitai-browser-plus#173]
### Describe the bug.

version 3.2.4 always return CivitAI Browser+: File: "xxxxxx.safetensors" not found on CivitAI servers, it looks like the file is not available for download.

--------------------------------------
same error without ARIA2 enable
Port TCP 24000 is open ( firewall & box )
Download link in description work fine

### Steps to reproduce the problem.

1. search for sdxl models/loras
2. Press download
3. log return : CivitAI Browser+: File: "xxxxxx.safetensors" not found on CivitAI servers, it looks like the file is not available for download.

### Expected behavior

Download start

### System info

* Extension version: 3.2.4
* OS: W11
* SD-WebUI version: v1.7.0-331-gcb5b335a (https://github.com/AUTOMATIC1111/stable-diffusion-webui/commit/cb5b335acddd126d4f6c990982816c06beb0d6ae)
* Python: 3.10.5

### Console logs

```Shell
CivitAI Browser+: File: "xxxxx.safetensors" not found on CivitAI servers, it looks like the file is not available for download.
CivitAI Browser+: Error occured during download of "xxxxx.safetensors"
```

**Comment by Lesteriax** [comment]
Having the same issue here

**Comment by BlafKing** [comment]
Thanks for reporting!

Currently working on this issue, CivitAI has limited the usage of API calls and this extension uses a single API which no longer works.

I'll post an update with a fix shortly!
Also working on fixing other bugs which is taking some time :,)

**Comment by ResidentChief** [comment]
Same issue here..

**Comment by BlafKing** [comment]
This issue should now be fixed in ([v3.2.5](https://github.com/BlafKing/sd-civitai-browser-plus/commit/c3129a7fd3d4cc63378027725e8fc67de15fdedb))!

Like I said above, a personal CivitAI API key is now required for downloading some models.
You can create your own key in your [CivitAI Account settings](https://civitai.com/user/account)
And afterwards, you can insert that API Key in the settings of CivitAI Browser+,
The settings is called `Personal CivitAI API key` located in SD-WebUI settings tab, and then under the CivitAI Browser+ tab.

I'll go ahead and close this bug report now since the issue should be fixed! :)

**Comment by ResidentChief** [comment]
Nice work chief!  Thanks heaps :)
